crates/goose/src/context_mgmt/auto_compact.rs

```diff
@@ -317,8 +317,8 @@ mod tests {
             id: "test_session".to_string(),
             working_dir: PathBuf::from(working_dir),
             description: "Test session".to_string(),
-            created_at: "2024-01-01T00:00:00Z".to_string(),
-            updated_at: "2024-01-01T00:00:00Z".to_string(),
+            created_at: Default::default(),
+            updated_at: Default::default(),
```

Comment:
this could cause flakiness in tests. probably they would catch something then though

Reply:
would it? DateTime::default() returns unix epoch 0, not now()